### PR TITLE
datatc Post

### PR DESCRIPTION
**Pull request by lokijuhy**
Blog post showcasing the data-file-interactions part of data traffic control. I decided to save the data-reproducibility part of data traffic control for a separate post to keep things digestible.

### CODE REVIEW

**Comment by lokijuhy** [comment]
Hello @iivankovic @arcsi1989 @mcmahom5 @matteobe - I put you all as reviewers on this post because I think you are all people who may give me some helpful feedback on this post! I'm not looking for all of you to review, but I'd appreciate a look-over if you have some time the next few days!

**Comment by mcmahom5** [comment]
Looks good to me!

File: content/post/datatc-intro/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+authors: 
+- laurakinkead
+date: 2021-06-22
+tags: 
+- Reproducibility
+- Data Science
+categories:
+- Research
+projects: []
+---
```
Comment (iivankovic) [inline content/post/datatc-intro/index.md:13]: Maybe to add a feature photo?

**Review by iivankovic — APPROVED** [review]
It's entertaining and reads very well! I would only add a feature photo if there's an appropriate one :)

**Pull request merged**